metadata/stream.yaml

```diff
@@ -0,0 +1,163 @@
+              sha256: e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855
+      packet:
+        release: 30.1.2.3
+        formats:
+          "raw.xz":
+            disk:
+              location: https://artifacts.example.com/Oofohng0xo2phai5.raw.xz
+              signature: https://artifacts.example.com/Oofohng0xo2phai5.raw.xz.sig
+              sha256: e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855
+      qemu:
+        release: 30.1.2.3
+        formats:
+          "qcow.xz":
+            disk:
+              location: https://artifacts.example.com/Siejeeb6ohpu8Eel.qcow.xz
+              signature: https://artifacts.example.com/Siejeeb6ohpu8Eel.qcow.xz.sig
+              sha256: e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855
+      virtualbox:
+        release: 30.1.2.3
+        formats:
+          ova:
+            disk:
+              location: https://artifacts.example.com/yohsh2haiquaeYah.ova
+              signature: https://artifacts.example.com/yohsh2haiquaeYah.ova.sig
+              sha256: e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855
+      vmware:
+        release: 30.1.2.3
+        formats:
+          ova:
+            disk:
+              location: https://artifacts.example.com/quohgh8ei0uzaD5a.ova
+              signature: https://artifacts.example.com/quohgh8ei0uzaD5a.ova.sig
+              sha256: e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855
+
+    images:
+      # Cloud images to be launched directly by users.  These are in a
+      # separate section because they might not always in sync with the
+      # release artifacts above.
+      aws:
+        regions:
+          us-east-1:
+            # We know the release because we uploaded it, so might as well
+            # list it.
+            release: 30.1.2.3
+            image: ami-0123456789abcdef
+          us-east-2:
+            release: 30.1.2.3
+            image: ami-0123456789abcdef
+      azure:
+        # We could give a specific image URN here, but we probably want
+        # users to always use a Marketplace URN.  So this is a static
+        # string, and represents advice rather than a value we might
+        # change.
+        image: Fedora:CoreOS:stable:latest
+      gcp:
+        # We could give a specific image name here, but we probably want
+        # users to always use an image family.  So this is a static string,
+        # and represents advice rather than a value we might change.
+        image: projects/fedora-cloud/global/images/family/fedora-coreos-stable
+      digitalocean:
+        # We don't control platform ingest, so an image slug is probably
+        # the best we can do.
+        image: fedora-coreos-stable
+      packet:
+        # Images don't have addressable versions, so an operating system
+        # slug is the best we can do.
+        image: fedora_coreos_stable
+
+    updates:
```

Comment:
I'm unsure about the indentation of this. I don't think we want to have different update metadata for each architecture, to avoid exploding releng complexity. If so, I think this can become a top-level section on its own.

Reply:
Yeah, in hindsight that choice seems somewhat arbitrary.  If we want different update targets for different arches, presumably we'd also want them for different platforms.  I don't want to introduce that level of complexity right now, so I'll switch this to a single value.